kubernetes/config/incluster_config_test.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+import unittest
+
+from kubernetes.client import configuration
+
+from .incluster_config import ConfigException, InClusterConfigLoader
+
+_TEST_PORT_ENV_NAME = "port"
```

Comment:
Consider reusing the names of the env vars defined as constants in the module under test (e.g. ``_SERVICE_HOST_ENV_NAME``) rather than defining new names in this test module.  Since the environ dict is passed to the function under test, there is no need to vary the names of the env vars and the names can be fixed across runtime and test.